src/ui/Analytics/Analytics.ts

```diff
@@ -504,6 +523,12 @@ export class Analytics extends Component {
     }
   }
 
+  private clearCookies() {
+    Cookie.erase('visitorId');
+    Cookie.erase('clientId');
+    Cookie.erase('visitId');
```

Comment:
The `visitId` is no longer tracked by the UI. It is determined by the UA service. This is something we can safely remove from the JSUI code-base in a dedicated PR.

Reply:
For GDPR reasons, should we still erase it if it exists though, let's say a client upgrade from a version where it was set, but then decides to allow the disabling analytics.

Maybe I'm being too paranoid here 😅

Reply:
I'll leave it for now